s4sim/hardware/config.py

```diff
@@ -883,13 +883,13 @@ def get_example():
 
     sfwhm = OrderedDict()
     sfwhm["LFS1"] = 72.8
```

Comment:
please add a comment above this to explain where these numbers are coming from

Reply:
Done